framework/audio/engine/3rdparty/soloud/include/soloud.h

```diff
@@ -80,13 +80,13 @@ freely, subject to the following restrictions:
 // Configuration defines
 
 // Maximum number of filters per stream
-#define FILTERS_PER_STREAM 8
+#define FILTERS_PER_STREAM 0

```

Comment:
this causes compiler warnings lelewere though

Reply:
Understood, I will fix it.